api/preprints/serializers.py

```diff
@@ -76,6 +77,10 @@ class PreprintSerializer(JSONAPISerializer):
     is_preprint_orphan = ser.BooleanField(read_only=True)
     license_record = NodeLicenseSerializer(required=False, source='license')
 
+    reviews_state = ser.CharField(read_only=True, max_length=15)
+
+    submit_for_review = ser.BooleanField(required=False, write_only=True)
```

Comment:
Hmm, having a general use field for when the user has pushed the actual save button would be fairly useful. Could we just do something along the lines of `submit`, `done`, `commit`? Pinging @brianjgeiger again.